### PR TITLE
Dw 1473 table sort fix

### PR DESCRIPTION
**Pull request by Deven-Rawat**

### CODE REVIEW

File: cms/src/main/resources/ckeditor/plugins/table/dialogs/table.js [hunk 1]
```diff
@@ -3,673 +3,667 @@
...
-                    td = thElems.getItem(i);
-                  }
-
-                  width = td.getStyle('width');
-                  if (width) {
-                    value += width.replace(/%/, "");
-                    if (i != elemsCount - 1) {
-                      value += ",";
+            if (!pass) {
+                alert("'Column with control' must be empty or coma separated non-0 numbers which sum is equal to 100 and number is equal to number of columns (ex. '80,20' for 2 columns or '30,30,40' for 3 columns)"); // jshint ignore:line
```
Comment (GeoffHayward) [inline cms/src/main/resources/ckeditor/plugins/table/dialogs/table.js:73]: Typo in the word 'coma'

**Review by GeoffHayward — COMMENTED** [review]
What are the  dialogs and table.js files? They are empty.

File: cms/src/main/resources/ckeditor/plugins/table/dialogs/table.js (hunk 1 above)
Comment (Deven-Rawat) [reply cms/src/main/resources/ckeditor/plugins/table/dialogs/table.js:73]: Done

**Comment by Deven-Rawat** [comment]
> What are the dialogs and table.js files? They are empty.

dialogs file is removed. and table.js modified

**Review by GeoffHayward — APPROVED** [review]

**Pull request merged**